lib/mapping.js

```diff
@@ -528,6 +528,12 @@ var options = {
       param: 'metric',
       info: 'set the metric used to calculate median for multiple runs tests [loadTime]'
     },
+    'medianRun': {
```

Comment:
Could you please rename it to `medianrun` all lower. This is used on command line where camel casing is not desired. The others (`name` and `api`) are fine as camel case.